src/status_im/ui/screens/profile/contact/views.cljs

```diff
@@ -18,6 +18,8 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(def ^:private nickname-max-length 24)
```

Comment:
It's good to have this var placed in `status-im2.constants` ns.

Additionally, we have an existing constants `profile-name-max-length` which holds the same value `24`. Not sure if this is the same.

Reply:
As mentioned by John [here](https://discord.com/channels/624634427930312714/928625369542713396/1151464141735870544). Max length is the same for both profile name & nickname. @smohamedjavid I have updated the code to use `profile-name-max-length` here too.

Reply:
Thank you @ajayesivan! 🚀